cmd/monitor/import.go

```diff
@@ -39,17 +36,27 @@ var monitorImportCmd = &cobra.Command{
 			log.Fatalf("Failed to connect Datadog API server: %s\n", err)
 		}
 
-		raw, err := ioutil.ReadFile(inputPath)
+		var monits []datadog.Monitor
+		for _, inputPath := range args {
+			var decoded []datadog.Monitor
+			raw, err := ioutil.ReadFile(inputPath)
+			if err != nil {
+				log.Fatalf("fatal: %s\n", err)
```

Comment:
This error message should be write more detail reason.

Reply:
For example, shoud I change this message to  `”Failed to read json file: %s\n"` ?

Reply:
> Error strings should not start with a capital letter because they'll often be prefixed before printing:
Ref: https://github.com/golang/go/wiki/Errors

If Golang rules follow, I think `failed to read JSON file: %s\n"` is better.